app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt

```diff
@@ -264,33 +263,17 @@ fun DashboardScreen(
 
                             val sources by vm.sources.collectAsStateWithLifecycle(initialValue = emptyList())
 
-                            Column(
-                                modifier = Modifier.fillMaxSize(),
-                            ) {
-                                sources.forEach {
-                                    BundleItem(
-                                        bundle = it,
-                                        onDelete = {
-                                            vm.delete(it)
-                                        },
-                                        onUpdate = {
-                                            vm.update(it)
-                                        },
-                                        selectable = bundlesSelectable,
-                                        onSelect = {
-                                            vm.selectedSources.add(it)
-                                        },
-                                        isBundleSelected = vm.selectedSources.contains(it),
-                                        toggleSelection = { bundleIsNotSelected ->
-                                            if (bundleIsNotSelected) {
-                                                vm.selectedSources.add(it)
-                                            } else {
-                                                vm.selectedSources.remove(it)
-                                            }
-                                        }
-                                    )
-                                }
-                            }
+                            BundleListScreen(
```

Comment:
Is this imported correctly? I've not checked in IDE but it looks like there are no imports for the screen

Reply:
I checked this compiles successfully and runs without any issues on AVD when I leave this PR, but maybe I can check this one more time

Reply:
I just checked for it, clean and rebuild project, and launch on AVD, seems no problem. Note that `DashboardScreen` and newly created `BundleListScreen` is in the same package `app.revanced.manager.ui.screen`, so theoretically should have no problem even without the `import` statement.